src/backend/database_migrations/versions/20221130_222651_mpx_hepc_support.py

```diff
@@ -0,0 +1,37 @@
+"""add MPX and HCV DB support for gisaid/genbank template
+
+
+Create Date: 2022-11-30 22:26:58.178142
+
+"""
+import enumtables  # noqa: F401
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20221122_222651"
+down_revision = "20221129_012949"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO aspen.pathogens (slug, name) VALUES ('HCV', 'Hepatitis C')
+        """
+    )
+
+    op.execute("select nextval('aspen.pathogen_repo_configs_id_seq')")
+    op.execute("select nextval('aspen.pathogen_repo_configs_id_seq')")
```

Comment:
```suggestion
    op.execute(
        """
        SELECT setval('aspen.pathogen_repo_configs_id_seq', (SELECT MAX(id) FROM aspen.pathogen_repo_configs)+1);
        """
    )
```

Reply:
It's more robust to use the current max val to update the sequence

Reply:
I agree that if we want contiguous ids, this is the better approach, but I'm not sure about the `+1` part. When the default insert pulls via `nextval` on the sequence, I think it might already have a +1 in the returned value from `nextval`. If @jgadling is confident this code is correct, then just go ahead, but if not, you might want to mess around on your local DB for a bit, verify there's no off-by-one-error.

Reply:
awesome thanks @jgadling for the suggestion